components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -250,14 +281,14 @@ export default class ChannelInviteModal<T extends Value> extends React.PureCompo
             inviteError = (<label className='has-error control-label'>{this.state.inviteError}</label>);
         }
 
-        const numRemainingText = (
-            <FormattedMessage
-                id='multiselect.numPeopleRemaining'
-                defaultMessage='Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. '
-                values={{
-                    num: MAX_SELECTABLE_VALUES - this.state.values.length,
-                }}
-            />
+        const header = (
+            <h1>
+                <FormattedMessage
```

Comment:
This is not a good change, this is going to break how you can "translate the string", in english, clearly you said "Add people to Whatever", but maybe in another language you would say "XXXX ChannelName XXXX" so... you can't assume the order of the words in an translations.

Reply:
@jespino That's the way it was.
I just changed New Members to people.

Previously it was:
"channel_invite.addNewMembers": "Add New Members to ",

I made it:
 "channel_invite.addNewMembers": "Add people to ",

Nothing else.

Reply:
Instead of using the <span> you can use the 'values' functionality of React Intl, and that would solve potential issues.

Reply:
Did that.